### DISCUSSION TITLE
MySQL Workbench

### DISCUSSION THREAD

**Discussion by alexxGmZ** [catppuccin/catppuccin#249] [Port Requests]
Kinda sad that there's little to none themes for MySQL Workbench. It would be nice if there's a catpuccin "code_editor.xml" for MySQL Workbench.

**Comment by unseen-ninja** [comment]
I've turned this into an issue #250 for easier tracking across port requests. c:
If you're potentially available to work on this please let is know.

**Comment by sgoudham** [comment]
Hey :wave:,

I've closed this discussion since the original post doesn't include any links on how to theme the application or link to any source code of other themes being applied to the application.

Would be more than happy to re-open once the post is updated with the relevant links or instructions on how to theme it. 

Thanks!
